Project.toml

```diff
@@ -3,14 +3,13 @@ uuid = "ced4e74d-a319-5a8a-b0ac-84af2272839c"
 version = "0.6.2"
 
 [deps]
-Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+ChainRules = "082447d4-558c-5d27-93f4-14fc19e9eca2"
```

Comment:
First, thanks a lot for all the suggestions! :)

ChainRules is needed for now, because we are using `chol_blocked_rev` here: https://github.com/TuringLang/DistributionsAD.jl/blob/5ec520116e3ff8b6f80534a3029674020c464843/src/common.jl#L11

Reply:
You're welcome! That makes sense. I added an alternate suggestion below.